source/agora/common/crypto/Key.d

```diff
@@ -430,3 +430,13 @@ unittest
     Point point_Address = Point(kp.address);
     assert(verify(point_Address, enroll_sig, "BOSAGORA"));
 }
+
+/// Test for converting from `Point` sturcture to `PublicKey` structure
+unittest
+{
+    import agora.common.crypto.Schnorr;
+
+    Pair pair = Pair.random();
+    auto pubkey = PublicKey(pair.V.data[]);
```

Comment:
Can't we do this?
```suggestion
    auto pubkey = PublicKey(pair.V[]);
```

Reply:
Yes. we can. It works. Thanks.

Reply:
> > Can't we do this?
> 
> Yes. we can. It works. Thanks.

I checked a little more
I think you need to fix this to test Point.opSlice.

Reply:
> I think you need to fix this to test Point.opSlice.

Do you mean I should change to calling the Point.opSlice instead of [] operator? It would be more helpful if you suggest a code.